### ISSUE TITLE
SDK Registration failed

### ISSUE THREAD

**Issue by KroSmi** [The1only/rosettadrone#198]
### Describe the bug

After i generate the APK file from android studio 
![image](https://github.com/RosettaDrone/rosettadrone/assets/149018172/a96cb82a-3cf4-419e-a441-f92a46c5a5bb)
and after installing the app on my android device i get this messages 
![image](https://github.com/RosettaDrone/rosettadrone/assets/149018172/97a6d923-70c4-4631-86b1-590d1a491d13)
![image](https://github.com/RosettaDrone/rosettadrone/assets/149018172/4aeec467-ec6a-4e10-b1cd-56b002ae60db)
the package name in the dji developers sdk key is correct (sq.rogue.rosettadrone)
And can someone tell me which steps should i take to connect my android device to my dji air 2s when i fix this registration problem?

### Drone Model

```Markdown
DJI air 2s
```

### Smartphone Model

```Markdown
- Model:
- Android version: 12.0
```

**Comment by KroSmi** [comment]
i still got the issue but i want to inform you that i have checked both API keys and they are both created and registered correctly. So the problem i am facing right now is that i don;t get the registration failed message, i only get the ''registering please wait..'' and when i connect the phone with the controller the app does nothing. the only button i am able to press is TEST MODE.

**Comment by alexbhandari** [comment]
If your API key is correctly placed in the AndroidManifest file, package name is correct, and you have internet access on the phone it should connect (did for me). Did you also fill the Google maps API key?

For connecting to the drone, yes you just connect the controller to the phone with USB cable and the controller to the drone as usual.

**Comment by KroSmi** [comment]
they are all correct but nothing is working :( . Can i get the apk from someone who has already has created the app ?

**Comment by 3iga** [comment]
I have a same issue. Did you find out a solution for it?

**Comment by 3iga** [comment]
> they are all correct but nothing is working :( . Can i get the apk from someone who has already has created the app ?
> 

I think, dji sdk is not supported in your region, because when i changed my region with vpn to france and created new account with new key, Rosetta started to work. However now i have problem with connection to qgs.

**Comment by alexbhandari** [comment]
@3iga good point regarding the region - didn't think of that. Connection is where I am getting stuck as well. Not sure how to get it to connect and have put more details here: https://github.com/RosettaDrone/rosettadrone/issues/201
